test/over_react/component_declaration/component_base_test.dart

```diff
@@ -1040,6 +1040,58 @@ main() {
       });
     });
 
+    group('UiStatefulComponent2', () {
+      TestStatefulComponent2Component statefulComponent;
+
+      setUp(() {
+        statefulComponent = new TestStatefulComponent2Component();
+        statefulComponent.state = JsBackedMap.from({'test': true});
+      });
+
+      group('`state`', () {
+        group('getter:', () {
+          test('returns a UiState view into the component\'s state map', () {
+            expect(statefulComponent.state, const TypeMatcher<TestStatefulComponent2State>());
```

Comment:
Just curious... why not use `isA< TestStatefulComponent2State >()` instead of `const TypeMatcher`?

Reply:
because i didn't write this test and just copy/pasted the tests above it and updated them to use UiComponet2. lol